app/helpers/filters_helper.rb

```diff
@@ -157,6 +165,51 @@ def applied_filters_count(filter_type)
     filters_count(applied_filters(filter_type))
   end
 
+  def check_your_answers_filters_list(session_filters, filter_type)
+    if filter_type == "lettings_logs"
+      check_your_answers_lettings_filters_list(session_filters)
+    else
+      check_your_answers_sales_filters_list(session_filters)
+    end
+  end
+
+  def check_your_answers_lettings_filters_list(session_filters)
+    [
+      { id: "years", label: "Collection year", value: formatted_years_filter(session_filters) },
+      { id: "status", label: "Status", value: formatted_status_filter(session_filters) },
+      { id: "needstype", label: "Needs type", value: formatted_needstype_filter(session_filters) },
+      { id: "assigned_to", label: "Assigned to", value: formatted_assigned_to_filter(session_filters) },
+      { id: "owned_by", label: "Owned by", value: formatted_owned_by_filter(session_filters) },
+      { id: "managed_by", label: "Managed by", value: formatted_managed_by_filter(session_filters) },
+    ]
+  end
+
+  def check_your_answers_sales_filters_list(session_filters)
+    [
+      { id: "years", label: "Collection year", value: formatted_years_filter(session_filters) },
+      { id: "status", label: "Status", value: formatted_status_filter(session_filters) },
+      { id: "assigned_to", label: "Assigned to", value: formatted_assigned_to_filter(session_filters) },
+      { id: "owned_by", label: "Owned by", value: formatted_owned_by_filter(session_filters) },
+      { id: "managed_by", label: "Managed by", value: formatted_managed_by_filter(session_filters) },
+    ]
+  end
```

Comment:
Are the lists of available filters for lettings and sales tied closely together elsewhere or defined fairly separately? 

I'm thinking that these are almost the same so it's tempting to de-duplicate a bit, but only if they are already fairly tied to being very similar.

Reply:
Yeah, at the moment the filter names seem to be getting set at the same place (I think the request would be coming through with the form params from _log_filters.html.erb so would likely be updated together), so I think it's probably a good shout to deduplicate here!